css-layout-classes/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link href="styles.css" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <h1>Choose your character!</h1>
+    </header>
+    <div class="container">
+      <div class="row">
+        <div class="column-one-third">
+          <img src="./images/link.png" alt="picture of link" class="image">
+        </div>
+        <div class="column-one-third">
+          <div class="column">
+            <h2>Super Smash Bros. Ultimate</h2>
+            <p>Super Smash Bros.Ultimate received universal acclaim from both critics and players, with some critics
+              calling it the best installment in the series. It received praise for its large amount of content and
+              fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.</p>
+          </div>
+        </div>
+        <div class="column-one-third">
+          <img src="./images/yoshi.png" alt="picture of yoshi" class="image">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-fourth">
+          <img src="./images/kirby.png" alt="picture of kirby" class="image">
+        </div>
+        <div class="column-one-fourth">
+          <img src="./images/fox.png" alt="picture of fox" class="image">
+        </div>
+        <div class="column-one-fourth">
+          <img src="./images/pikachu.png" alt="picture of pikachu" class="image">
+        </div>
+        <div class="column-one-fourth">
+          <img src="./images/ness.png" alt="picture of ness" class="image">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column">
+          <h2>Donkey Kong</h2>
+          <p>Donkey Kong is a series of video games featuring the adventures of a gorilla called Donkey Kong, conceived by
+            Shigeru Miyamoto in 1981. The franchise consists mainly of two game genres, but also includes additional
+            spin-off titles of various genres.</p>
+          <p>The games of the first genre are the mostly single-screen platform/action puzzle types, featuring Donkey Kong
+            as the opponent in an industrial construction setting.</p>
+        </div>
+        <img src="./images/dk.png" alt="image of dk" class="image">
```

Comment:
This `column` and `image` element should have a column class like your other column classes, that describe the styling they apply.